packages/utils/src/__tests__/url-ops.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { isExternalLink } from '../url-ops';
+
+describe('isExternalLink', () => {
+    it('should return true if the link is external', () => {
```

Comment:
@yauheni-deriv Test case description doesn't match the test, Maybe better to add separate test cases.